services/java/capture/.devcontainer/devcontainer.json

```diff
@@ -29,7 +29,6 @@
 		"redhat.fabric8-analytics",
 		"vscjava.vscode-spring-initializr",
 		"vscjava.vscode-spring-boot-dashboard",
-		"vscjava.vscode-java-pack",
```

Comment:
Any reason to remove this?

Reply:
I was getting errors every time I rebuilt the container saying that:

Cannot activate:
'Test Runner for Java' extension
Language Support for Java by Red Hat 

Far from being missing from the "extensions" array "vscjava.vscode-java-pack" was entered twice. It also appeared that this pack contains the Test Runner for Java and Language Support for Java by Red Hat. I removed what appeared to be this duplicate entry to see if adding a duplicate extension was causing the errors to be reported. It made no difference.